ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -522,6 +522,7 @@ bool UncoupledProcessesTimeLoop::setCoupledSolutions()
                 const std::size_t c_id =
                     std::distance(_per_process_data.begin(), found_item);
 
+                MathLib::LinAlg::setLocalAccessibleVector(*_process_solutions[c_id]);
```

Comment:
Is that necessary? Isn't that already taken care of by the nonliner solver or by setInitialConditions() ?